packages/client/src/checkout/types/patchCheckoutOrderItem.types.ts

```diff
@@ -0,0 +1,13 @@
+import type { Config } from '../../types';
+
+export type CheckoutOrderItemUpdateRequest = {
```

Comment:
Should we have a better name for this? Something like `PatchCheckoutOrderItemData`?

Reply:
✅ I initially went with the api spec definition, but `PatchCheckoutOrderItemData` sounds good to me